sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditFragment.java

```diff
@@ -119,16 +121,24 @@
 
 	private List<Item> caseConfirmationBasisList;
 
+	private boolean isDifferentJurisdiction;
```

Comment:
Rename to "differentJurisdiction" (I suggest to not name boolean fields starting with "is" - the boolean type already suggests that, and it makes naming getters and setters pretty problematic; we had issues with that in the past)